config.js

```diff
@@ -36,8 +36,9 @@ exports.updateFromArgv = function() {
         .demand('apikey')
         .describe('username', 'Specify the username of the bot, for example GnuGo')
         .describe('apikey', 'Specify the API key for the bot')
-        .describe('greeting')
+        .describe('greeting', 'Greeting message to appear in chat at first move (ex: -Hello, have a nice game-)')
```

Comment:
Shouldn't this be `--greeting -Hello, have a nice game-`. I find it confusing to leave out the argument name, but start the example with a dash anyway.

Reply:
i dont think so, in https://github.com/wonderingabout/gtp2ogs/blob/greeting-botcommand/docs/OPTIONS-LIST.md#greeting we provide `"Hello, have a nice game"` as an example of greeting message

we are not trying to give an example of how to use the gtp2ogs argument, just an example of messages bot admin could use

it is also the same for farewell https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#farewell

Reply:
we could remove the examples but i think new bot admins would find them helpful

Reply:
yes i'm okay with a sample, but please use `Hello, have a nice game` instead of `-Hello, have a nice game-`.